central/sensor/service/connection/connection_impl.go

```diff
@@ -339,7 +341,7 @@ func (c *sensorConnection) processIssueLocalScannerCertsRequest(ctx context.Cont
 		err = errors.New("requestID is required to issue the certificates for the local scanner")
 	} else {
 		var certificates *storage.TypedServiceCertificateSet
-		certificates, err = localscanner.IssueLocalScannerCerts(namespace, clusterID)
+		certificates, err = securedclustercertgen.IssueLocalScannerCerts(namespace, clusterID)
```

Comment:
Is this a mistake?

Reply:
No, I renamed the `central/localscanner/certificates.go` to `central/securedclustercertgen/certificates.go` and I added another function to it (IssueSecuredClusterCerts).